contracts/domainRegister.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+contract DomainRegister {
+    struct DomainInfo {
```

Comment:
line 5 
it is a little bit over-logically, to use struct for this case
// - 1 point